docs/admin/live-restore.md

```diff
@@ -0,0 +1,74 @@
+<!--[metadata]>
++++
+title = "Keep containers alive during daemon downtime"
+description = "How to keep containers running when the daemon isn't available."
+keywords = ["docker, upgrade, daemon, dockerd, live-restore, daemonless container"]
+[menu.main]
+parent = "engine_admin"
+weight="6"
++++
+<![end-metadata]-->
+
+# Keep containers alive during daemon downtime
+
+By default, when the Docker daemon terminates, it shuts down running containers.
+Starting with Docker Engine 1.12, you can configure the daemon so that containers remain
+running if the daemon becomes unavailable. The live restore option helps reduce
+container downtime due to daemon crashes, planned outages, or upgrades.
+
+## Enable the live restore option
+
+There are two ways to enable the live restore setting to keep containers alive
+when the daemon becomes unavailable:
+
+* When you start the Docker daemon, pass the `--live-restore` flag:
+
+    ```bash
+    $ sudo dockerd --live-restore
+    ```
+
+* If the daemon is already running and you don't want to stop it, you can add the configuration to the daemon configuration file. For example, on a linux system the default configuration file is `/etc/docker/daemon.json`.
+
+Use your favorite editor to enable the `live-restore` option in the
+`daemon.json`.
+
+```bash
+{
+"live-restore": true
+}
+```
+
+You have to send a `SIGHUP` signal to the daemon process for it to reload the
+configuration. For more information on how to configure the Docker daemon using
+config.json, see [daemon configuration file](../reference/commandline/dockerd.md#daemon-configuration-file)
+
+## Live restore during upgrades
+
+The live restore feature supports restoring containers to the daemon for upgrades from one minor release to the next. For example from Docker Engine 1.12.1 to 1.13.2.
```

Comment:
Did we test this scenario? Does it keep the containers running when upgrading (e.g. `apt-get install`)